test/e2e/autoscale_test.go

```diff
@@ -320,7 +320,7 @@ func assertAutoscaleUpToNumPods(ctx *testContext, numPods int32) {
 		ctx.t.Fatalf("Error during scale up: %v", err)
 	}
 
-	if err := assertNumberOfPods(ctx, minPods, maxPods); err != nil {
+	if err := assertNumberOfPods(ctx, numPods, maxPods); err != nil {
```

Comment:
I think you'll want to adjust the comments below in the `TestAutoscaleUpCountPods` function

Reply:
thanks for pointing that out.
done.